eth/tracers/api.go

```diff
@@ -615,25 +632,80 @@ func (api *API) traceBlock(ctx context.Context, block *types.Block, config *Trac
 			}
 		}()
 	}
+
+	IOdump := "TransactionIndex, Incarnation, VersionTxIdx, VersionInc, Path, Operation\n"
+	RWstruct := []state.DumpStruct{}
 	// Feed the transactions into the tracers and return
 	var failed error
 	blockCtx := core.NewEVMBlockContext(block.Header(), api.chainContext(ctx), nil)
+
+	london := api.backend.ChainConfig().IsLondon(block.Number())
+
 	for i, tx := range txs {
+		// copy of statedb
+		statedb := statedb.Copy()
```

Comment:
could we make this changes optional? to not influence original API and performance? maybe add a flag to the request or something?